tests/functions.test.ts

```diff
@@ -23,6 +21,38 @@ async function extractUser() {
     mode: "FUNCTIONS"
   })
 
+  const user: User = await client.chat.completions.create({
+    messages: [{ role: "user", content: "Jason Liu is 30 years old" }],
+    model: "gpt-3.5-turbo",
+    response_model: UserSchema
+  })
+
+  return user
+}
+
+async function extractUserValidated() {
+  const UserSchema = z.object({
+    age: z.number(),
+    name: z
+      .string()
+      .refine(name => name === name.toUpperCase(), {
+        message: "Name must be uppercase, please try again"
+      })
+      .describe("The users name, all uppercase")
```

Comment:
```suggestion
      .describe("The users name, all uppercase")
```

the point is that the validation fails and the llm fixes it, not that we tell the llm the right answer